app/templates/learnergroups.hbs

```diff
@@ -34,15 +34,20 @@
       {{input type='text' value=titleFilter placeholder=placeholderValue}}
     </div>
   </div>
-  <section class='grouplist'>
-    <h2>{{title}}</h2>
+
+  <section class='grouplist clearfix'>
```

Comment:
Instead of using the section class clearfix use the sass @include clearfix; on the element.